lib/components/AwardHero/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { selectActiveAward, useSelector } from "@/lib/redux";
+import Link from "next/link";
+import { BreadCrumb } from "primereact/breadcrumb";
+import { Button } from "primereact/button";
+
+export interface ComponentProps {
```

Comment:
This type of component props is only required for page components, as that's how next.js passes the internal server props to the page controller. In this case a simpler component props might be better. Try something like the following:

```ts
export interface ComponentProps {
  id: string;
}

export default function AwardsHero(props: ComponentProps) {
  const {id} = props; 
  ...

  return (...);
}

// in the component usage just pass the id as a prop
...
<AwardsHero id={id}/>
```

Reply:
forgot to add id as component prop. Added a new commit with that change